docs/how_to_install_collector.md

```diff
@@ -0,0 +1,50 @@
+How to install collector command
+=====================================
+
+The collector command can run in setup-hw container and bare metal linux server. 
+in this memo describe how to install bare metal server.
+
+## 1.Clone code from github repository
+
+```
+$ git clone https://github.com/cybozu-go/setup-hw
+```
+
+## 2.Build & install collector command
+
+```
+$ cd setup-hw
+$ make install
+```
+
+## 3.Setup config
+
+Put a bmc-user.json file in /etc/neco/ that must have "support" user due to use the collector command.
+
+```
+{
+  "support": {
+    "password": {
+      "raw": "cybozu"
```

Comment:
I don't think it's a good idea to put our company name as a password example here, even though this is not a real password.
There are 2 reasons.

1. Some people may misunderstand they should place exactly `"cybozu"` here. We should clarify that the string is just for example.
2. This is a document for OSS users, so it would be good to use organization-neutral examples.

```suggestion
      "raw": "raw password here"
```